src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -119,17 +119,17 @@ const noJobsMessage = div({ style: { fontSize: 20, margin: '1rem' } }, [
         ...Utils.newTabLinkProps,
         href: 'https://support.terra.bio/hc/en-us/articles/360027920592',
       },
-      ['What is a job?']
+      ['What is a submission?']
```

Comment:
This article is "How to troubleshoot failed workflows".  Is that the correct article to link here?  And if so, is "What is a submission" (or job) the correct text?

Reply:
It does seem like sort of a strange choice for text linking to that article, but that feels out of scope for this PR. I think changing from "job" to "submission" there is appropriate.

Reply:
Agreed on both counts. We don't seem to have an article explaining the concepts and their relationships, more so "steps to do X".

Reply:
Nice find... I was searching for "Terra submission monitoring" 🤦‍♂️